test/e2e/test.utils.js

```diff
@@ -60,7 +60,7 @@ export const config = {
     COMMAND_TIMEOUT: 2 * 60 * 1000,
   },
   testnet: {
-    userKeyName: 'user2',
+    userKeyName: 'user1',
```

Comment:
Why user1? Addresses and mnemonics are of user2 here 🤔
Does our test config is setup according to user1 only?

Reply:
You can choose any name for your key. I use `user1` for both local and testnet testing to maintain consistency.

Reply:
PS: using the `agd keys add` command.